worlds/stardew_valley/logic/logic.py

```diff
@@ -401,7 +400,9 @@ def __init__(self, player: int, options: StardewValleyOptions):
         self.registry.item_rules.update(self.registry.tree_fruit_rules)
         self.registry.item_rules.update(self.registry.seed_rules)
         self.registry.item_rules.update(self.registry.crop_rules)
+
         self.registry.item_rules.update(self.mod.item.get_modded_item_rules())
+        self.mod.item.append_vanilla_item_rules(self.registry.item_rules)  # New regions and content means new ways to obtain old items
```

Comment:
Just reading this line, without checking the content of the method, I wouldn't know what it does. So the name is not very good.
Suggestion: `modify_vanilla_item_rules_with_mod_additions`